_includes/posts_block.html

```diff
@@ -37,6 +37,9 @@ <h3 class="post-title"><a href="{{ post.url | relative_url }}" rel="bookmark">{{
         </div><!-- .post-card-inside -->
       </article><!-- .post -->
       {% endfor %}
+      <p class="block-buttons center">
+        <a href="/blog/" class="button white large">More Videos</a>
```

Comment:
There must be a way to address `blog` dynamically, but I can't find it. For our purposes, this is totally valid.